### ISSUE TITLE
Session stolen when connecting on the same user

### ISSUE THREAD

**Issue by lostallmymoney** [stascorp/rdpwrap#1581]
Hi, so the issue I'm having which makes it weird to use RDP while I am using the desktop is that it steals the current session and transfers it to the 1st rdp connection. RDP wrap still works though as I can then connect on the desktop and have multiple sessions. This can be a problem because I have scripts to run when connecting on desktop vs RDP. I also find it to meddle with RDPWrap's goals. I'm sure there's a way to figure this out, I am not the only one this fix could be useful to. Thanks.

**Comment by AltF5** [comment]
This is typical RDP (Terminal Services) behavior

**Comment by lostallmymoney** [comment]
Yes, still feels like an issue tho. I don't think I am qualified for the job but.. I think we could trick terminal services into not transfering the session over like it does with a 3rd device? Maybe start an empty dummy session that would get transfered/stop normal behavior.
Or make terminal services think we're already in rdp ?
Anyways you are probably more knowledgeful. Thanks
________________________________

**Comment by AltF5** [comment]
Not sure how deep that plumbing goes. Sounds like something you want is TS Shadowing ( https://docs.microsoft.com/en-us/troubleshoot/windows-server/remote/shadow-terminal-server-session )... something I haven't tried much since leveraging Jump Desktop, VNC, or TeamViewer for such cases.
